build.sbt

```diff
@@ -7,6 +7,17 @@ import sbtrelease.ReleasePlugin.ReleaseKeys.releaseProcess
 import sbtrelease.ReleaseStateTransformations._
 import sbtrelease.Utilities._
 import sbtunidoc.Plugin.UnidocKeys._
+import ScoverageSbtPlugin._
+
+lazy val scoverageSettings = Seq(
+  ScoverageKeys.coverageMinimum := 60,
+  ScoverageKeys.coverageFailOnMinimum := false,
+  ScoverageKeys.coverageHighlighting := {
+    if (scalaBinaryVersion.value == "2.10") false
```

Comment:
This is super nitpicky, but I've always thought of `if (cond) false else true`-ish stuff as a code smell. Why not just `scalaBinaryVersion.value != "2.10"`?